front-end/src/app/shared/models/report-types/f24-report.model.ts

```diff
@@ -0,0 +1,46 @@
+import { plainToClass, Transform } from 'class-transformer';
+import { Report } from './report.model';
+import { LabelList } from '../../utils/label.utils';
+import { BaseModel } from '../base.model';
+
+export enum F24FormTypes {
+  F24N = 'F24N',
+  F24A = 'F24A',
+}
+
+export type F24FormType = F24FormTypes.F24N | F24FormTypes.F24A;
+
+export const F24FormTypeLabels: LabelList = [
+  [F24FormTypes.F24N, 'FORM 24'],
+  [F24FormTypes.F24A, 'FORM 24'],
+];
+
+export const F3xFormVersionLabels: LabelList = [
+  [F24FormTypes.F24N, 'Original'],
+  [F24FormTypes.F24A, 'Amendment'],
+];
+
+export class F24 extends Report {
+  override form_type = F24FormTypes.F24A;
+
+  committee_name: string | undefined;
```

Comment:
Do you think it's worth putting the address and treasurer fields in the common report class?

Reply:
I'm not against it.  They're certainly common to both F3x and F24.  Looking ahead at [what we currently have for Form specs](https://docs.google.com/spreadsheets/d/18TtYCHWjj0gbsubJXQTbALEkbK_t93t0/edit#gid=211201010), _most_ of what I'm seeing has these fields.  I do think there's a pretty good argument for adding them to the Report class.

Reply:
Done!